### PR TITLE
Adding HTTP quickstarts for Python and .NET (C#)

### PR DESCRIPTION
**Pull request by paulyuk**
The maintainers prefer we have HTTP quickstarts to go along with SDK quickstarts.

This contributes the following:
- Pub-sub python-http
- Pub-sub csharp-http (.net 6 minimal code)
- Many consistency improvements

### CODE REVIEW

**Comment by paulyuk** [comment]
@amulyavarote

File: pub_sub/python-http/checkout/app.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+base_url = os.getenv('BASE_URL', 'http://localhost') + ':' + os.getenv(
+                    'DAPR_HTTP_PORT', '3500')
+PUBSUB_NAME = 'order_pub_sub'
+TOPIC = 'orders'
+logging.info('Publishing to baseURL: %s, Pubsub Name: %s, Topic: %s' % (
+            base_url, PUBSUB_NAME, TOPIC))
+
+while True:
+    order = {'orderid': random.randint(1, 1000)}
```
Comment (amulyavarote) [inline pub_sub/python-http/checkout/app.py]: I assume we are keeping all small letters for python and keeping camel case for other languages?

Comment (paulyuk) [reply pub_sub/python-http/checkout/app.py]: That's a bug good catch.  Fixing.

Comment (paulyuk) [reply pub_sub/python-http/checkout/app.py]: Fixed

**Pull request merged**